apps/api/src/app/analytics/analytics.controller.ts

```diff
@@ -0,0 +1,22 @@
+import { Body, Controller, Logger, Post, UseGuards } from '@nestjs/common';
+import { SkipThrottle } from '@nestjs/throttler';
+import { AnalyticsService, UserAuthGuard, UserSession } from '@novu/application-generic';
+import { IJwtPayload } from '@novu/shared';
+
+@Controller({
+  path: 'analytics',
```

Comment:
Would suggest calling this "telemetry" and the endpoint "measure" as some blockers will look for "track" keywords

Reply:
Good to know, didn't realise those blockers were so aggressive. I agree, we should definitely make this change.